packages/anon-aadhaar-pcd/test/pcd.test.ts

```diff
@@ -4,44 +4,54 @@ import { init, prove, verify } from '../src/pcd'
 import { assert } from 'chai'
 import { genData } from './utils'
 import { ArgumentTypeName } from '@pcd/pcd-types'
-import { WASM_URL, ZKEY_URL, VK_URL } from '../src/constants'
+import { sha256Pad } from '@zk-email/helpers/dist/shaHash'
+import { Uint8ArrayToCharArray } from '@zk-email/helpers/dist/binaryFormat'
+import { VK_URL, WASM_URL, ZKEY_URL, splitToWords } from '../src'
 
 describe('PCD tests', function () {
   this.timeout(0)
 
   let testData: [bigint, bigint, bigint, bigint]
+  let paddedMsg: Uint8Array
+  let messageLen: number
 
   this.beforeAll(async () => {
-    testData = await genData('Hello world', 'SHA-1')
+    const signedData = 'Hello-world'
+
+    testData = await genData(signedData, 'SHA-256')
+    ;[paddedMsg, messageLen] = sha256Pad(
+      Buffer.from(signedData, 'ascii'),
+      512 * 3
+    )
   })
 
   it('PCD flow location prover', async function () {
-    const dirName = __dirname + '/../artifacts/RSA'
+    const dirName = __dirname + '/../../circuits/artifacts'
     const pcdInitArgs: PCDInitArgs = {
-      wasmURL: dirName + '/main.wasm',
+      wasmURL: dirName + '/qr_verify.wasm',
       zkeyURL: dirName + '/circuit_final.zkey',
-      vkeyURL: dirName + '/verification_key.json',
+      vkeyURL: dirName + '/vkey.json',
       isWebEnv: false,
     }
 
     await init(pcdInitArgs)
 
     const pcdArgs: AnonAadhaarPCDArgs = {
```

Comment:
create pcdArgs look like so complicate. Is it okay if we create function create pcdArgs since we know the type of all field.